gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -279,11 +285,26 @@ private GoogleCloudStorageItemInfo getObjectMetadata(StorageResourceId resourceI
         : new IOException("Error reading " + resourceId, exception);
   }
 
+  private void recordGCSAPILatency(
+      MeasureLong measure, long time, String status, String method, String protocol) {
+    if (storageOptions.isTraceLogEnabled()) {
+      Map<String, Object> jsonMap = new HashMap<>();
+      jsonMap.put("method", method);
+      jsonMap.put("status", status);
+      jsonMap.put("protocol", protocol);
+      jsonMap.put("duration", time);
```

Comment:
The method has latency and this is named duration. Please keep this consistent with what we already have and name the elapsed?

Reply:
I will make it consistent it with latency.